frontend/src/views/StatsPage.vue

```diff
@@ -14,7 +23,7 @@
       marginwidth="0"
       aria-labelledby="visits"
     />
-    <h3 id="channels">Canaux d'acquisition</h3>
+    <h3 class="text-left my-3">Canaux d'acquisition</h3>
```

Comment:
```suggestion
    <h3 id="channels" class="text-left my-3">Canaux d'acquisition</h3>
```
pour le `aria-labelledby`

Reply:
Ce n'est plus d'actualité depuis qu'on fait un lien vers le dashboard externe : https://github.com/betagouv/ma-cantine/pull/413/commits/e46be6048ea4a642c3de6253cf48b4ff95d2b98b